feat(hyprland): add mirrorOf member of Monitor

Dr0med4r opened the pull request:
For this, it is necessary to use 'monitors all' instead of 'monitors.' 
If this is not done, the 'monitoradded' event will add an empty monitor, and the 'monitorremoved' event will fail with a null error. 
This fixes issues with mirrored displays, but now mirrored monitors are listed under 'monitors' (which they previously were, but not fully and I don't know if this is intended behaviour).

Dr0med4r committed:
feat: add mirrorOf member of Monitor
For this it is necessary to use "monitors all" and not "monitors"
if this is not done the monitoradded event will add an empty monitor and
the monitorremoved event will fail with a null error.

Aylur commented on lib/hyprland/monitor.vala:
```diff
@@ -25,6 +25,7 @@ public class AstalHyprland.Monitor : Object {
     public bool vrr { get; private set; }
     public bool actively_tearing { get; private set; }
     public bool disabled { get; private set; }
+    public string mirror_of {get; private set; }
```
This should be a `Monitor` type

Aylur commented on lib/hyprland/monitor.vala:
```diff
@@ -49,6 +50,7 @@ public class AstalHyprland.Monitor : Object {
         vrr = obj.get_boolean_member("vrr");
         actively_tearing = obj.get_boolean_member("activelyTearing");
         disabled = obj.get_boolean_member("disabled");
+        mirror_of = obj.get_string_member("mirrorOf");
```
should handle if it is `null` since it was added in `0.45` this breaks backwards compatibility

Dr0med4r committed:
add backwards compatablity for mirrorOf
This is null if the monitor is not mirrored or mirrorOf is not yet
implemented else it is the monitor mirrored.
